src/vivarium_gates_shigella_vaccine/paths.py

```diff
@@ -1,6 +1,6 @@
 from pathlib import Path
 
-from vivarium_gates_shigella_vaccine.globals import PROJECT_NAME
```

Comment:
I prefer to import the entire globals module when we use it.  Help's distinguish project globals from any module defined globals.